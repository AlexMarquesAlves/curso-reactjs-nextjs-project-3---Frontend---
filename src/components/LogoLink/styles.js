import styled, { css } from 'styled-components';

export const Container = styled.a`
  ${({ theme }) => css`
    display: flex;
    align-itens: center;
    text-decoration: none;
    color: inherit;

    > img {
      max-height: 3rem;
    }
  `}
`;
