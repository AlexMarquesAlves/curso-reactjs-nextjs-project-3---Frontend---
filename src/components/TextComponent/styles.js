import styled from 'styled-components';

export const TextComponent = styled.div``;
