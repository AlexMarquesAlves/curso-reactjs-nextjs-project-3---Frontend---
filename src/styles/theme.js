export const theme = {
  colors: {
    primaryColor: '#0a1128',
    secondaryColor: '#dc143c',
    white: '#fff',
  },
  fonts: {},
  spacings: {},
};
