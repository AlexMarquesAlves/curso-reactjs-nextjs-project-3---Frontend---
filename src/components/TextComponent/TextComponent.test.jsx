import { screen } from '@testing-library/react';
import { renderTheme } from '../../styles/render-theme';
import { TextComponent } from '.';

describe('<TextComponent />', () => {
  it('should render', () => {
    renderTheme(<TextComponent>Children</TextComponent>);
    expect(screen.getByRole('heading')).toBeInTheDocument();
  });
});
