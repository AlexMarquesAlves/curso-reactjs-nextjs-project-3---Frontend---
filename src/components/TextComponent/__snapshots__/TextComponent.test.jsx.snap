// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TextComponent /> should match snapshot 1`] = `
.c0 {
  font-size: 2.4rem;
}

<p
  class="c0"
>
  Children
</p>
`;
